cpp/src/prims/detail/sample_and_compute_local_nbr_indices.cuh

```diff
@@ -1568,9 +1566,6 @@ uniform_sample_and_compute_local_nbr_indices(
   bool with_replacement)
 {
   using edge_t = typename GraphViewType::edge_type;
```

Comment:
This line's now failing a style check.  It wants the `=` signs aligned without the intervening lines.

Reply:
Ran clang-format.